### DISCUSSION TITLE
Port issue

### DISCUSSION THREAD

**Discussion by Castolo** [EasyEngine/easyengine#1720] [Support Requests]
When I run the command "`ee site create [domainname.com] --wp --letsencrypt`" is sends "`Cannot create/start proxy container. Please make sure port 80 and 443 are free.`" What should I do?

**Comment by amsteus** [comment]
I have the exact same issue. Disabled ufw. Allowed the ports as below. Still the same issue.

sudo ufw --force disable
Firewall stopped and disabled on system startup
 ee site create website.com --type=wp --admin-user=name --admin-pass=password --ssl=le
Error: Cannot create/start proxy container. Please make sure port 80 and 443 are free.

sudo ufw allow 80
Rules updated
Rules updated (v6)
sudo ufw allow 443
Rules updated
Rules updated (v6)
ee site create website.com --type=wp --admin-user=username --admin-pass=password --ssl=le
Error: Cannot create/start proxy container. Please make sure port 80 and 443 are free.
